doc/howtos/rdtraining/13_inheritance.rst

```diff
@@ -0,0 +1,244 @@
+``models/inherited_model.py``.
+
+.. exercise:: Add fields on Users
+
+    - Add the following field on ``res.users``:
+
+    ===================== ======================================================
+    Field                 Type
+    ===================== ======================================================
+    property_ids          One2many inverse of ``user_id`` on ``estate.property`` 
+    ===================== ======================================================
+
+    - Add a domain on the field to list only the available properties.
+
+Now let's add the field in the view to check eveything is working well!
+
+View Inheritance
+================
+
+**Reference**: the documentation related to this topic can be found in
+:ref:`reference/views/inheritance`.
+
+.. note::
+
+    **Goal**: at the end of this section:
+
+    The list of available properties linked to a salesperson is displayed on the user form view
+
+    .. image:: 13_inheritance/media/users.png
+        :align: center
+        :alt: Users
+
+Instead of modifying existing views in place (by overwriting them), Odoo
+provides view inheritance where children 'extension' views are applied on top of
+root views, and can add or remove content from their parent.
+
+An extension view references its parent using the ``inherit_id`` field, and
+instead of a single view its ``arch`` field is composed of any number of
+``xpath`` elements selecting and altering the content of their parent view:
+
+.. code-block:: xml
+
+    <record id="inherited_model_view_form" model="ir.ui.view">
+        <field name="name">inherited.model.form.inherit.test</field>
+        <field name="model">inherited.model</field>
+        <field name="inherit_id" ref="inherited.inherited_model_view_form"/>
+        <field name="arch" type="xml">
+            <!-- find field description and add the field
+                 new_field after it -->
+            <xpath expr="//field[@name='description']" position="after">
+              <field name="new_field"/>
+            </xpath>
+        </field>
+    </record>
+
+``expr``
+    An XPath_ expression selecting a single element in the parent view.
+    Raises an error if it matches no element or more than one
+``position``
+    Operation to apply to the matched element:
+
+    ``inside``
+        appends ``xpath``'s body at the end of the matched element
+    ``replace``
+        replaces the matched element with the ``xpath``'s body, replacing any ``$0`` node occurrence
+        in the new body with the original element
+    ``before``
+        inserts the ``xpath``'s body as a sibling before the matched element
+    ``after``
+        inserts the ``xpaths``'s body as a sibling after the matched element
+    ``attributes``
+        alters the attributes of the matched element using special
+        ``attribute`` elements in the ``xpath``'s body
+
+When matching a single element, the ``position`` attribute can be set directly
```

Comment:
That's... misleading. In view inheritance we can only ever match a single element, that gives the impression that xpath can match and update several. In fact this should be the subject of a note or warning.

Anyway the second version is mostly a shortcut of sorts (I'm not sure it's even a good idea to add it), the semantics are not significantly different from xpath's.